packages/js-split-provider/src/lib/js-split-provider.ts

```diff
@@ -21,17 +21,18 @@ export class OpenFeatureSplitProvider implements FeatureProvider {
     });
   }
   
-  async isEnabled(flagId: string, defaultValue: boolean, context?: Context): Promise<boolean> {
-    return this.getBooleanValue(flagId, defaultValue, context);
+  async isEnabled(flagId: string, defaultValue: boolean, context: Context, options?: FlagEvaluationOptions): Promise<boolean> {
+    return this.getBooleanValue(flagId, defaultValue, context, options);
   }
 
   /**
    * Split doesn't directly handle booleans as treatment values.
    * It will be up to the provider author and it's users to come up with conventions for converting strings to booleans.
    */
-  async getBooleanValue(flagId: string, defaultValue: boolean, context?: Context): Promise<boolean> {
+  async getBooleanValue(flagId: string, defaultValue: boolean, context: Context, options?: FlagEvaluationOptions): Promise<boolean> {
     await this.initialized;
-    const stringValue = this.client.getTreatment(context?.userId ?? 'anonymous', flagId);
+    // simply casting Context to Attributes is likely a bad idea.
```

Comment:
Will this be handled in a follow up PR?

Reply:
Ya, was going to handle it with some of the upcoming Context research.